python/mxnet/kvstore.py

```diff
@@ -290,19 +309,21 @@ def num_workers(self):
         return size.value
 
     def save_optimizer_states(self, fname):
-        """Saves optimizer (updater) state to file.
+        """Saves the optimizer (updater) state to a file.
+
+        This is often used when checkpointing the current progress of training a module.
```

Comment:
Do we need a one-line paragraph? Can this be part of the first line?

Also it makes more sense to say we checkpoint the model, not that we checkpoint "the progress of a model" agree/disagree?